build.gradle

```diff
@@ -9,6 +20,15 @@ plugins {
     id 'org.jenkins-ci.jpi' version '0.43.0'
 }
 
+def internalRepoHost = System.getenv("SNPS_INTERNAL_ARTIFACTORY")
+
+repositories {
+    maven { url "https://sig-repo.synopsys.com/bds-integrations-release" }
+    mavenCentral()
+    maven { url 'https://plugins.gradle.org/m2/' }
```

Comment:
Note: These 3 repositories have been explicitly added here instead of re-using them from `"https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-repositories.gradle"`. This is because for a reason not 100% clear to me, the Gradle wrapper on this project is unable to resolve the repositories when they are included using the `apply from` syntax outside the buildscript closure. 
Open to any insights on this but the current syntax makes Gradle happy and it seemed a minor redundancy to optimize, hence moving forward with this syntax itself.